.github/workflows/detect-breaking-change.yml

```diff
@@ -14,8 +14,7 @@ jobs:
     - uses: gradle/gradle-build-action@v3
       with:
         cache-disabled: true
-        # The 2.14.0 is already reporting breaking changes (see please https://github.com/opensearch-project/OpenSearch/issues/13308) 
-        arguments: japicmp -Djapicmp.compare.version=2.14.0-SNAPSHOT
+        arguments: japicmp
```

Comment:
@Bukhtawar it addresses one of your concerns regarding unreleased changes, we sadly had to postpone the fix till 2.14.0 release because there were breaking changes in 2.14.0 already (comparing to 2.13.0)